### DISCUSSION TITLE
Build error occurred: fetch is not found globally and no fetcher passed

### DISCUSSION THREAD

**Discussion by arimendelow** [vercel/next.js#13786] [Help] [answered]
I have this issue specifically when attempting to deploy on Heroku - when deploying on Vercel, it works as expected.

This can be replicated by trying to deploy the below on Heroku:
[github repo with project](https://github.com/arimendelow/rj-website)

_Originally posted by @arimendelow in https://github.com/vercel/next.js/issues/12794#issuecomment-639195722_

**Comment by jamesmosier** [comment, marked as answer]
Hi @arimendelow. Could you try upgrading Next to the latest stable version (`9.4.4`)

The fix mentioned in the issue you linked came out after the initial `9.4.0` release which you have in your `package.json` in your project.

**Reply by arimendelow** [reply]
This worked! I thought I was on the latest release as running `npm update` next didn't do anything, but manually changing the version number in my `package.json` forced the update.

Thanks @jamesmosier !
